README.md

```diff
@@ -1,4 +1,9 @@
-# README
+# Avalon Media System
+[Avalon Media System](http://www.avalonmediasystem.org) is an open source system for managing large collections of digital audio and video. The project is led by the libraries of [Indiana University](http://www.iu.edu) and [Northwestern University](http://www.northwestern.edu) with funding in part by a three-year National Leadership Grant from the [Institute of Museum and Library Services](http://www.imls.gov).
```

Comment:
Does the funding line need changing to list all grant funding or maybe leave that whole clause out of this paragraph for inclusion at the end of the README?